src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -113,15 +119,36 @@ public virtual Task Init(INethermindApi nethermindApi)
             return Task.CompletedTask;
         }
 
+        private void MigrateSecondsPerSlot()
+        {
+            ulong defaultValue = _blocksConfig.GetDefaultValue<ulong>(nameof(IBlocksConfig.SecondsPerSlot));
+            if (_blocksConfig.SecondsPerSlot != _mergeConfig.SecondsPerSlot)
+            {
+                if (_blocksConfig.SecondsPerSlot == defaultValue)
+                {
+                    _blocksConfig.SecondsPerSlot = _mergeConfig.SecondsPerSlot;
+                }
+                else if (_mergeConfig.SecondsPerSlot == defaultValue)
+                {
+                    _mergeConfig.SecondsPerSlot = _blocksConfig.SecondsPerSlot;
```

Comment:
I find this going both ways somewhat disturbing. `MergeConfig.SecondsPerSlot` shouldn't be used anymore, right?

Reply:
That is correct, but i followed the same think we are doing with with `MiningConfig` and `BlockConfig` . you can check it here.
https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Init/Steps/MigrateConfigs.cs